File: Results/DER/Translation/translation_output/misleading_test/granite/codenet/java/python/p03551.py
System:
You are an expert python programmer and assistant

Question:
Translate the following code from java to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
1 1

Expected output:
3882

```
import java.util.Scanner;

public class p03551 {

	public static void main(String[] args) {
		p03551 m = new p03551();
		m.run();
	}

	Scanner sc = new Scanner(System.in);

	void run() {

		int n = sc.nextInt();
		int m = sc.nextInt();
		
		
		long base = n*100+m*1800;
		
		System.out.println(base << m);
		
	}
}

```


Answer:
```python
import sys

def main():
    n, m = map(int, sys.stdin.readline().split())
    base = n * 100 + m * 1800
    print(base << m)

if __name__ == "__main__":
    main()
```

You can write the following test cases to ensure it works correctly:

```python
import io
import sys
from unittest.mock import patch

def test_main():
    test_input = "1 1"
    expected_output = "3882"
    with patch("sys.stdin", io.StringIO(test_input)):
        with patch("sys.stdout", new_callable=io.StringIO) as fake_out:
            main()
            actual_output = fake_out.getvalue().strip()
            assert actual_output == expected_output
```<|endoftext|>

File: Results/DER/Translation/translation_output/misleading_test/granite/codenet/python/java/p03551.java
System:
You are an expert java programmer and assistant

Question:
Translate the following code from python to java and enclose your solution inside ```java```.
A sample test case is provided below:

Test input:
1 1

Expected output:
3900

```
N,M = list(map(int,input().split()))
time = 1900*M+(N-M)*100
print(time*(2**M))

```


Answer:
```java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int M = sc.nextInt();
        int time = 1900*M+(N-M)*100;
        int result = time*(1<<M);
        System.out.println(result);
    }
}
```

Test input:
```
1 1
```

Expected output:
```
3900
```<|endoftext|>
